spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/storage/AzureStorageAutoConfiguration.java

```diff
@@ -56,7 +55,7 @@ public void collectTelemetry() {
     @Bean
     @ConditionalOnMissingBean
     public CloudStorageAccount storage(ResourceManagerProvider resourceManagerProvider,
-            AzureStorageProperties storageProperties) {
+                                       AzureStorageProperties storageProperties) {
```

Comment:
Could you remove this format change?

Reply:
we do not use default format ?

Reply:
what's default format? we could discuss the format later. But in this commit, it's irrelevant with log change.

Reply:
It has been done from my IDE when file changes. Retrieved.